contracts/rewards/src/contract/execute.rs

```diff
@@ -77,113 +77,106 @@ where
     ) -> Result<(), ContractError> {
         let cur_epoch = self.current_epoch(block_height)?;
 
+        let event =
+            self.load_or_store_event(event_id, contract_addr.clone(), cur_epoch.epoch_num)?;
+
+        self.store
+            .load_epoch_tally(contract_addr.clone(), event.epoch_num)?
+            .unwrap_or(EpochTally::new(
+                contract_addr,
+                cur_epoch,
+                self.store.load_params().params,
+            ))
+            .record_participation(worker)
+            .then(|mut tally| {
+                if matches!(event, LoadState::New(_)) {
+                    tally = tally.increment_event_count()
+                }
```

Comment:
I think we can get rid of this and just set the event count to 1 in the `unwrap_or` a few lines above.

Reply:
no, because the tally has to be incremented based on if the event is new, not if the tally is new

Reply:
oh yes, ok nvm